### ISSUE TITLE
FEATURE:  mask overlay

### ISSUE THREAD

**Issue by pepi-55** [CyberTimon/RapidRAW#868]
### Feature Description

The **red mask overlay** is nice, but it is not very fine-tuned. If possible make a short key for it to show and hide it. now you have to switch to the basic panel when finetuning the opacity etc. With a shortkey for instance the letter "M" you can turn it of and on and see what the effect is when editing  or using the opacity of the mask. 
Its a pain in the ass that when editing the opacity slider you cannot see the real effect because of the mask overlay.
The eye button is nice but it turns the mask  off.

### Similar feature requests

- [x] I have searched existing issues and discussions and didn't find a duplicate

**Comment by CyberTimon** [comment]
Thanks. Hovering over the visibility toggle for mask containers or mask components now hides the red overlay and reveals the actual changes beneath the mask, with the mask visibility either enabled or disabled - making comparisons much easier: https://github.com/CyberTimon/RapidRAW/commit/61bc82042d59de8516e7449298fcec76d2cfb051

Will appear in the next release.

https://github.com/user-attachments/assets/7ea32911-d247-40d4-bed7-af0e845a1618
